Code/Mobile_Android/app/src/main/java/fr/keyz/damageDetails/DamageDetailsViewModel.kt

```
@@ -0,0 +1,106 @@
+                        leaseId =  _currentDamage.value!!.leaseId,
+                        damageId = _currentDamage.value!!.id,
+                        updateDamageInput = updateInput
+                    )
+                    _currentDamage.value = _currentDamage.value?.copy(
+                        fixStatus = DamageStatus.PLANNED,
+                        fixPlannedAt = OffsetDateTime.ofInstant(Instant.ofEpochMilli(date), ZoneOffset.UTC),
+                    )
+                }
+            } catch (e: ApiCallerServiceException) {
+                _apiError.value = e.getCode()
+            } catch (e : Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)